js/initcontext.go

```diff
@@ -370,17 +371,18 @@ func (i *InitContext) allowOnlyOpenedFiles() {
 
 func getInternalJSModules() map[string]interface{} {
 	return map[string]interface{}{
-		"k6":             k6.New(),
-		"k6/crypto":      crypto.New(),
-		"k6/crypto/x509": x509.New(),
-		"k6/data":        data.New(),
-		"k6/encoding":    encoding.New(),
-		"k6/execution":   execution.New(),
-		"k6/net/grpc":    grpc.New(),
-		"k6/html":        html.New(),
-		"k6/http":        http.New(),
-		"k6/metrics":     metrics.New(),
-		"k6/ws":          ws.New(),
+		"k6":                    k6.New(),
+		"k6/crypto":             crypto.New(),
+		"k6/crypto/x509":        x509.New(),
+		"k6/data":               data.New(),
+		"k6/encoding":           encoding.New(),
+		"k6/execution":          execution.New(),
+		"k6/experimental/redis": redis.New(),
```

Comment:
I really do prefer if this just imports https://github.com/grafana/xk6-redis and makes `New` from there.

In order to not have it also add `k6/x/redis` you should probably move the actual module code in a subpackage and have the top level module just register it a kin to how it's done in [xk6-websockets](https://github.com/grafana/xk6-websockets) for example. 

This way if we decide to update it (which is really likely) we will just be bumping the version in the go.mod.

Reply:
Great, I'll do that then, thanks a lot for the input 👯 🕺🏻

Reply:
Implemented your recommendation 👍🏻 

The xk6-redis extension is now imported by the `go.mod`. The `k6/experimental/redis` module now calls `github.com/grafana/xk6-redis/redis.New` instead. I've removed the code initially imported in the experimental folder. I've kept but modified the README file, which is now inside of the experimental folder, and structured, so we can document all the experimental modules in there.